### ISSUE TITLE
Add CA root to Insomnia

### ISSUE THREAD

**Issue by gagaXD** [FiloSottile/mkcert#249]
Hello,

After running a `mkcert -install` and generating certificates with it, It works fine on Chrome & Firefox.

But, when I'm trying to send a request with [insomnia](https://insomnia.rest/), It tells me that the certificate can not be trusted.

Is there any way to specify to mkcert other "db" where the rootCA must be installed with `certutil` ?

Thanks for this great package !

**Comment by francoism90** [comment]
@gagaXD Did you find a solution? I'm also keep getting the same issue with Insomnia.

**Comment by alouini333** [comment]
You can disable certificate validation in the settings of Insomnia

**Comment by FiloSottile** [comment]
I'd be happy to add it to the supported DBs if someone can find out where the Insomnia's root store is.

**Comment by FinlayDaG33k** [comment]
Under the hood, Insomnia uses `Curl` with it's CA stored in your `TEMP`.
For me, this was `C:\Users\Finlay\AppData\Local\Temp\insomnia_2020.4.2\2017-09-20.pem`.  
I don't know where it is on Linux or Mac but you can find it by making a request and looking at the timeline:
![image](https://user-images.githubusercontent.com/13424273/99596863-55518980-29f7-11eb-85c6-2c9db379eac8.png)

Hopefully this information will be of use :)

**Comment by bNobo** [comment]
> Under the hood, Insomnia uses `Curl` with it's CA stored in your `TEMP`. For me, this was `C:\Users\Finlay\AppData\Local\Temp\insomnia_2020.4.2\2017-09-20.pem`. I don't know where it is on Linux or Mac but you can find it by making a request and looking at the timeline: ![image](https://user-images.githubusercontent.com/13424273/99596863-55518980-29f7-11eb-85c6-2c9db379eac8.png)
> 
> Hopefully this information will be of use :) I don't know how to "include" additional certificates but maybe someone else knows.

Yes, you can edit the file with a text editor and add your certificates into it. I tried and it works. When the file exists Insomnia will use it directly. So it is a viable workaround but I think you'll have to re-edit the file each time Insomnia is updated.

**Comment by leocavalcante** [comment]
Aha! Thank you very much for the hack.

I just noticed there is a `CApath`, would be nice if Insomnia sets this path so we can just place our custom CA certs there instead of editing the one in temp.

**Comment by ghislain-bernard** [comment]
Hi,

For latest versions 2022, the option to use the file 2017.09.20.pem is not possible anymore... :( and now it is impossible to use my own certificates from my private CA.

![image](https://user-images.githubusercontent.com/37534566/187907019-e6931137-5384-495b-b24e-b0613d4fba37.png)

![image](https://user-images.githubusercontent.com/37534566/187907062-6af251e1-3bbe-41bd-a150-5f86885953d0.png)

To use my private CA for server certificates, I disable the global option "Validate certificates" but it is not a good option.

![image](https://user-images.githubusercontent.com/37534566/187907268-87392512-2dee-4815-8fb2-e971deba2070.png)

I hope to use the CA system in future versions.

**Comment by laszlolaszlo** [comment]
It should be very good to able to use own Root CA in Insomnia. Thank you.
